app/views/rapidfire/question_groups/_question_group.html.haml

```diff
@@ -19,8 +19,6 @@
   .survey__admin-row__col.survey__admin-row__col--actions
     .course-list__col
       = link_to "Edit", edit_question_group_path(question_group)
-    .course-list__col
```

Comment:
This is the cloning link for a QuestionGroup, not a Survey. This should be kept.

Reply:
Okay, thanks for the update, let me correct that right away